velox/docs/functions/spark/decimal.rst

```diff
@@ -54,6 +54,13 @@ Decimal Functions
     Return the unscaled bigint value of a short decimal ``x``.
     Supported type is: SHORT_DECIMAL.
 
+.. spark:function:: unaryminus(x: decimal(p, s)) -> r: decimal(p, s)
```

Comment:
nit: move 'unaryminus'  before 'unscaled_value' to follow the alphabetical order.

Reply:
Updated, thanks